content/docs/setup/kubernetes/additional-setup/mesh-expansion/index.md

```diff
@@ -175,49 +196,115 @@ The following example shows accessing a service running in the Kubernetes cluste
 1.  First, on the cluster admin machine get the virtual IP address (`clusterIP`) for the service:
 
     {{< text bash >}}
-$ kubectl -n bookinfo get svc productpage -o jsonpath='{.spec.clusterIP}'
-10.55.246.247
+    $ kubectl get svc productpage -o jsonpath='{.spec.clusterIP}'
+    10.55.246.247
     {{< /text >}}
 
 1.  Then on the mesh expansion machine, add the service name and address to its `etc/hosts` file. You can then connect to
     the cluster service from the VM, as in the example below:
 
     {{< text bash >}}
-$ sudo echo "10.55.246.247 productpage.bookinfo.svc.cluster.local" >> /etc/hosts
-$ curl productpage.bookinfo.svc.cluster.local:9080
+$ echo "10.55.246.247 productpage.default.svc.cluster.local" | sudo tee -a /etc/hosts
+$ curl -v productpage.default.svc.cluster.local:9080
+< HTTP/1.1 200 OK
+< content-type: text/html; charset=utf-8
+< content-length: 1836
+< server: envoy
 ... html content ...
     {{< /text >}}
 
+The `server: envoy` header indicates the traffic indeed is intercepted by sidecar.
+
 ## Running services on a mesh expansion machine
 
-You add VM services to the mesh by configuring a
-[`ServiceEntry`](/docs/reference/config/networking/v1alpha3/service-entry/). A service entry lets you manually add
+1. We first setup a HTTP server on the VM instance, serving HTTP traffic on port 8080.
+
+    {{< text bash >}}
+    $ gcloud compute ssh ${GCE_NAME}
+    $ python -m SimpleHTTPServer 8080
+    {{< /text >}}
+
+1. Determine the VM instance IP address. For example, commands to find GCE instance IP are as below.
+
+    {{< text bash >}}
+    $ export GCE_IP=$(gcloud --format="value(networkInterfaces[0].networkIP)" compute instances describe  ${GCE_NAME})
+    $ echo ${GCE_IP}
+    {{< /text >}}
+
+1. Configure Service Discovery for VM by `ServiceEntry`. You add VM services to the mesh by configuring a
```

Comment:
Please update the entire text of the step to avoid clunky content for our users.
```suggestion
1. Configure a service entry to enable service discovery for the VM. You can add VM services to the mesh using a 
    [service entry](/docs/reference/config/networking/v1alpha3/service-entry/). Service entries let you manually add
    additional services to Pilot's abstract model of the mesh. Once VM services are part of the mesh's abstract model,  
    other services can find and direct traffic to them. Each service entry configuration contains the IP addresses, ports, 
    and appropriate labelsof all VMs exposing a particular service, for example:
```

Reply:
Adopted. but do we want to break the word `ServiceEntry`, which is a specific terminology in Istio?

Reply:
Yes, we do. See:  https://preliminary.istio.io/about/contribute/style-guide/#use-consistent-capitalization for details.